mir-ci/mir_ci/interfaces/benchmarkable.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC, abstractmethod
+from mir_ci.cgroups import Cgroup
+
+class Benchmarkable(ABC):
+    @abstractmethod
+    async def get_cgroup(self) -> Cgroup:
+        pass
+
+    @abstractmethod
+    async def __aenter__(self):
+        pass
+
+    @abstractmethod
+    async def __aexit__(self, *args):
+        pass
```

Comment:
Should make these mandatory:

```suggestion
class Benchmarkable(ABC):
    @abstractmethod
    async def get_cgroup(self) -> Cgroup:
        raise NotImplementedError

    @abstractmethod
    async def __aenter__(self):
        raise NotImplementedError

    @abstractmethod
    async def __aexit__(self, *args):
        raise NotImplementedError
```